test/metabase/api/preview_embed_test.clj

```diff
@@ -147,6 +147,52 @@
             (embed-test/test-query-results
              (mt/user-http-request :crowberto :get 202 (str (card-query-url card {:_embedding_params {:venue_id "enabled"}})
                                                             "?venue_id=200")))))))))
+(deftest default-value-card-query-test
```

Comment:
I'm not sure if this test was totally necessary, because preview_embed stuff reuses embed functions. But I added it anyway to be extra safe.

Reply:
I think it's good to have it, the fact that preview_embed uses embed functions is an implementation detail, we should have tests for the API layer regardless.